RebarTools.py

```diff
@@ -190,6 +190,32 @@ def Activated(self):
         MainBeamReinforcement.CommandBeamReinforcement()
 
 
+class BillOfMaterialTool:
+    def GetResources(self):
+        return {
+            "Pixmap": os.path.split(os.path.abspath(__file__))[0]
+            + "/icons/dropdown_list/BOM.svg",
+            "MenuText": QT_TRANSLATE_NOOP(
+                "RebarAddon", "Rebar Bill Of Material"
+            ),
+            "ToolTip": QT_TRANSLATE_NOOP(
+                "RebarAddon", "Generate Rebars Bill Of Material",
+            ),
+        }
+
+    def IsActive(self):
+        if FreeCADGui.ActiveDocument:
+            return True
+        else:
+            return False
```

Comment:
```suggestion
       return True if FreeCADGui.activeDocument() else False
```

Please update this everywhere